docs/release-notes/8.1.0-1.md

```diff
@@ -0,0 +1,18 @@
+# Percona XtraBackup 8.1.0-1 (2023-10-)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
```

Comment:
To set the expecations right,I would suggest users to use PXB 8.0 for 8.0 series. Also mention that PXB 8.1 will not take backups of 8.0 servers.